.github/workflows/release.yml

```diff
@@ -48,12 +48,18 @@ jobs:
                     echo "has-changesets=true" >> "$GITHUB_OUTPUT"
                   fi
 
+    notify-approval-needed:
+        name: Notify Slack - Approval Needed
+        needs: check-changesets
+        if: needs.check-changesets.outputs.has-changesets == 'true'
+        uses: ./.github/workflows/notify-slack-release-approval.yml
+
     version-bump:
         name: Bump versions and commit to main
-        needs: check-changesets
+        needs: [check-changesets, notify-approval-needed]
```

Comment:
This job won't run if the notification fails to send, which probably isn't desirable